chart/ci/default-values.yaml

```diff
@@ -0,0 +1,32 @@
+# This is the default values for the chart to run in CI system and it can be used as
+# a base template for other *-values.yaml files. Only resource requests are nullified
+# to allow starting the stack on github action runner and prevent issues related
+# to pod scheduling due to insufficient host resources.
+
+timescaledb-single:
```

Comment:
I think we also need the environment variables to disable telemetry here.

Reply:
Good catch! I added now.